11-node-frameworks/classworks/Gloria_Loi/index.js

```diff
@@ -0,0 +1,49 @@
+const Koa = require('koa');
+const serve = require('koa-static');
+const Router = require('koa-router');
+const session = require('koa-generic-session');
+const bodyParser = require('koa-bodyparser');
+const Joi = require('@hapi/joi');
+
+const router = Router();
+
+const schema = Joi.object({
+  firstname: Joi.string().min(3).required(),
+  lastname: Joi.string().min(3).required(),
+  text: Joi.string().min(3).required(),
+  mark: Joi.number(),
+});
+
+const users = [];
+let user;
+
+router.get('/get', async (ctx) => {
+  const sess = session;
```

Comment:
keep in mind that counter should work for any request to the server, even posting form. Split counting and getting the "session counter"

Reply:
basically it should be middleware which works for each request to the server, without any response